cmd/ip-masq-agent/ip-masq-agent.go

```diff
@@ -240,6 +246,8 @@ func validateCIDR(cidr string) error {
 func (m *MasqDaemon) syncMasqRules() error {
 	// make sure our custom chain for non-masquerade exists
 	m.iptables.EnsureChain(utiliptables.TableNAT, masqChain)
+	// make sure our custom chain for ipv6 non-masquerade exists
+	m.ip6tables.EnsureChain(utiliptables.TableNAT, masqChain)
```

Comment:
As per the discussion -- should we call EnsureChain if there is no ipv6 IPs?
(This applies to the other calls to ip6tables as well